core/src/driver/scheduler/system.rs

```diff
@@ -37,28 +37,21 @@ impl<'a> SystemScheduler<'a> {
         }
     }
 
-    fn start_solving_in_thread<'b>(
-        &self,
-        batch_id: BatchId,
-        solver_deadline: Instant,
-        scope: &Scope<'b>,
-    ) where
-        'a: 'b,
-    {
-        let driver = self.driver;
+    fn start_solving_in_background(&self, batch_id: BatchId, solver_deadline: Instant) {
+        let driver = self.driver.clone();
         let earliest_solution_submit_time = self
             .auction_timing_configuration
             .earliest_solution_submit_time;
-        scope.spawn(move |_| {
+        async_std::task::spawn(async move {
```

Comment:
Does this task need to be `await`-ed anywhere or is it automatically driven by the `async_std` runtime?

Reply:
It's like spawning a thread so it doesn't need to be awaited. It can be awaited through a join handle but it doesn't have to.